examples/features/advancedtls/README.md

```diff
@@ -0,0 +1,26 @@
+# gRPC Advanced Security Examples
+This repo contains example code for different security configurations for grpc-go using `advancedtls`.
+
+The servers run a basic echo server with the following setups:
+* Port 8885: A server with a good certificate using certificate providers and crl providers.
+* Port 8884: A server with a revoked certificate using certificate providers and crl providers.
+* Port 8883: A server running using InsecureCredentials.
+
+The clients are designed to call these servers with varying configurations of credentials and revocation configurations.
+* mTLS with certificate providers and CRLs
+* mTLS with custom verification
+* mTLS with credentials from credentials.NewTLS (directly using the tls.Config)
+* Insecure Credentials
+
+## Generate the credentials used in the examples
+Run `./generate.sh` from `/path/to/grpc-go/examples/features/advancedtls` to generate the `creds` directory containing the certificates and CRLs needed for these examples.
```

Comment:
I assume this isn't actually necessary since there is a `creds` directory in this PR?  If so, maybe move this section to the bottom and update the text here saying this is how you can do this if you need to make changes or something.

Reply:
Discussed, moved the bottom as a developer note